src/GuestConfiguration/GuestConfiguration.Test/ScenarioTests/GetVmGuestPolicyStatusTests.ps1

```diff
@@ -32,8 +32,8 @@ Get guest configuration policy report by Vm name scope for custom policy
 #>
 function Get-AzVMGuestPolicyStatus-VmNameScope_Custom
 {
-	$rgName = "amits-test"
-	$vmName = "amits-winser6"
+	$rgName = "aashishCustomrole7ux"
```

Comment:
same comment - users should not have to update these names whenever they re-record these tests

Reply:
I will create a new PR to update how tests have been written for GCRP PowerShell cmdets.
Wont fix in this PR.